src/utils/index.js

```diff
@@ -38,6 +39,17 @@ export const buildSchema = (customNodes = {}, customMarks = {}) => {
 		...customMarks,
 	};
 
+	/* Overwrite defaultOptions with custom supplied nodeOptions */
+	Object.keys(nodeOptions).forEach((nodeKey) => {
+		const nodeSpec = schemaNodes[nodeKey];
+		if (nodeSpec) {
+			schemaNodes[nodeKey].defaultOptions = {
```

Comment:
Or just `nodeSpec.defaultOptions = {...}`

Reply:
I'm not sure I follow. Could you elaborate?

Reply:
Ah, I only mean that we already have a shorter way to say `schemaNodes[nodeKey].defaultOptions = {...}` here. Not a big deal.